Neos.Neos/Classes/Aspects/NodeTypeConfigurationEnrichmentAspect.php

```diff
@@ -79,6 +77,27 @@ protected function addLabelsToNodeTypeConfiguration($nodeTypeName, array &$confi
         if (isset($configuration['properties'])) {
             $this->setPropertyLabels($nodeTypeName, $configuration, $declaredSuperTypes);
         }
+
+        if (isset($configuration['childNodes'])) {
+            $this->setChildNodeLabels($nodeTypeName, $configuration, $declaredSuperTypes);
+        }
+    }
+
+
+    /**
+     * @param $nodeTypeName
+     * @param array $configuration
+     * @param array $declaredSuperTypes
+     * @return void
+     */
+    protected function setChildNodeLabels($nodeTypeName, array &$configuration, array $declaredSuperTypes)
+    {
+        $nodeTypeLabelIdPrefix = $this->generateNodeTypeLabelIdPrefix($nodeTypeName);
+        foreach ($configuration['childNodes'] as $childNodeName => &$childNodeConfiguration) {
+            if ($this->shouldFetchTranslation($childNodeConfiguration)) {
```

Comment:
```suggestion
            if ($childNodeConfiguration && $this->shouldFetchTranslation($childNodeConfiguration)) {
```

Nodetype settings are always nullable in subtypes f.e.. So we always have to catch this case. Currently this would lead to an exception and make it impossible to overwrite a child node from a subtype.

Reply:
Thx for catching this! Comitted a fix where I also removed an unnecessary function argument